src/main/java/fi/dy/masa/tweakeroo/util/InventoryUtils.java

```diff
@@ -873,6 +902,34 @@ public static void swapElytraWithChestPlate(@Nullable PlayerEntity player)
         }
     }
 
+    private static double getArmorAndArmorToughnessValue(ItemStack stack, double base, AttributeModifierSlot slot)
+    {
+        final double[] total = {base};
+
+        stack.applyAttributeModifier(slot, (entry, modifier) -> {
+            if (entry.getKey().orElseThrow() == EntityAttributes.GENERIC_ARMOR
+                || entry.getKey().orElseThrow() == EntityAttributes.GENERIC_ARMOR_TOUGHNESS)
+            {
```

Comment:
Can you verify that this functions, because RegistryEntry<> is not the same as a RegistryKey<>

Reply:
Of course I could test this, but I figured I would ask.

Reply:
I tested it with a diamond chestplate and a netherite one, it works fine. From my experience, the RegistryEntry usually are not direct entries if you are getting them from world.getRegistryManager() and from packet/nbt codec, they have registry keys.

Reply:
Yes, I really don't like the registry stuff.  All of these terms are always confusing, and I personally wouldn't have done it this way; but if you say it works ?

Reply:
I think it is fine. Although it makes writing mods more painful, it avoids hard coding, and always reminds you to be compatible with datapacks.